static/js/settings_account.js

```diff
@@ -104,6 +104,36 @@ exports.set_up = function () {
         }
     });
 
+    function send_pass_reset() {
+        var email = $('#email_value').text().trim();
```

Comment:
Hmm, this doesn't feel like the right way to communicate this request to the server. We're gathering up an email address from what's currently in the form and asking the server to send that address a password reset link for the appropriate user... but in fact, we're already logged in as a specific user. So we should be asking the server to do a password reset for the logged-in user.

This probably involves some server-side changes to add an appropriate view function.